src/hooks/auth/useAuth.ts

```diff
@@ -45,9 +45,11 @@ export type UseAuthReturn = {
 const useAuth = (): UseAuthReturn => {
   const [user, setUser] = useRecoilState(appStore.user)
   const { connect, disconnect } = useConnection()
-  const { setCurrentUser } = useSendbirdChat()
+  const { setCurrentUser, features } = useSendbirdChat()
```

Comment:
this should be done in useNotificationRegister

Reply:
And we should default to on once the user grants permission. And if the user turns off, degister from sendbird, remove from firestore, remove async storage, and also remove permission

Reply:
Now the token register is running in two functions.
useNotificationRegister hook and connect function of useConnection hook.

useConnection is a library of sendbird, when connect call, token is automatically registered.
so, autoRegistration was changed to false where connect hook is used.

and it is already trying to register in useNotificationRegister when the app is running. useNotificationRegister is used even when the value changes in Setting.